modules/videoio/test/test_v4l2.cpp

```diff
@@ -58,6 +60,29 @@ struct Format_Channels_Depth
     float mul_height;
 };
 
+#ifdef DUMP_CAMERA_FRAME
+static std::string v4l2_fourcc_string(uint32_t fourcc)
```

Comment:
There are two functions: `fourccToStringSafe` and `fourccToString` declared in test_precomp.hpp, maybe they can be reused?

Reply:
Done.